packages/react-native-sdk/src/components/LocalVideoView.tsx

```diff
@@ -56,29 +63,59 @@ export const LocalVideoView = (props: LocalVideoViewProps) => {
   const isCameraOnFrontFacingMode = useStreamVideoStoreValue(
     (store) => store.isCameraOnFrontFacingMode,
   );
+  const pan = useRef(new Animated.ValueXY()).current;
```

Comment:
some comments of what this variable does and the animation logic would help for maintenance